public/locales/en-US/send.ftl

```diff
@@ -117,3 +126,30 @@ passwordIsSet = Password set
 maxPasswordLength = Maximum password length: { $length }
 # A short status message shown when there was an error setting the password
 passwordSetError = This password could not be set
+pageHeaderCredits = from the makers of Firefox
```

Comment:
Why is this starting lowercase? It would definitely benefit from a note explaining it.